tsconfig.json

```diff
@@ -2,11 +2,12 @@
   "compilerOptions": {
```

Comment:
I would recommend adding `"outDir": "./dist"` to `compilerOptions`, like we do for Apollo Client [here](https://github.com/apollographql/apollo-client/blob/610293d091f006e8044ae9853822bb96cb261756/tsconfig.json#L18), so that the compiled files don't end up mixed together with the `src/*` files. Then you can add `dist/` to `.gitignore` to avoid seeing all the compilation artifacts every time you run `git status`.

Reply:
I pushed up a commit that includes `ts-loader` so running `npm run build` will now catch those errors. From what I understand, we don't need to run `tsc` to compile as Webpack is doing that for us (now) so I don't believe we need an `outDir`. TBH I've wanted to chuck this laptop out the window *multiple* times with configuring TS + Babel + Webpack. I *think* it's working now but it's some kind of dark magic.